client/src/Components/TodoForm.jsx

```diff
@@ -1,103 +1,79 @@
-import React, { useState, useEffect, useRef } from 'react'
-import axios from "axios";
+import React, { useState, useEffect, useRef } from 'react';
+import axios from 'axios';
 
 function TodoForm(props) {
-    const API_URL = process.env.REACT_APP_API_URL || "http://localhost:5000";
-    const [input, setInput] = useState('');
-    const [placeholderInput, setPlaceholderInput] = useState('');
+  const API_URL = process.env.REACT_APP_API_URL || 'http://localhost:5000';
+  const [input, setInput] = useState(props.edit ? props.edit.name : '');
+  const [titleInput, setTitleInput] = useState(props.edit ? props.edit.title : '');
```

Comment:
using titleInput instead of "placeholdeInput"
Also... using setting to values in `props.edit` in cases where this is an edit form instead of a new item form.